core/db/migrate/20180404235352_add_default_status_to_reimbursements.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDefaultStatusToReimbursements < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default(:spree_reimbursements, :reimbursement_status, Spree::Reimbursement::PENDING)
```

Comment:
We should use the value of the constant here so we do not rely on the code may change in the future.